jdk.sh

```diff
@@ -16,9 +16,9 @@ JDK_PLATFORM=linux
 [[ $ARCHITECTURE != osx* ]] || JDK_PLATFORM=osx
 
 if [[ $JDK_PLATFORM == osx ]]; then
-  URL="https://cdn.azul.com/zulu/bin/zulu12.2.3-ca-jdk12.0.1-macosx_x64.tar.gz"
```

Comment:
Do we really need JAVA on macOS? This will work only for x86. I suggest we simply disable the dependency there.